bin/si-gen-cloud-control/src/pipeline-steps/generateAssetFuncs.ts

```diff
@@ -40,7 +40,7 @@ export function generateAssetFuncs(specs: PkgSpec[]): PkgSpec[] {
 
     const assetFunc: FuncSpec = {
       name: assetFuncName,
-      uniqueId: assetFuncUniqueKey,
+      uniqueId: assetFuncUniqueKey!,
```

Comment:
We should avoid this assertions generally. If it's optional, fail if not found. See comment above though